BlueprintUI/Sources/Environment/Keys/ViewCacheKey.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+extension Environment {
+    private enum ViewCacheKey: EnvironmentKey {
+        static let defaultValue: TypeKeyedCache? = nil
+    }
+
+    private static let fallback = TypeKeyedCache()
```

Comment:
I'm thinking we shouldn't have a static fallback.

Reply:
Yeah if we can get away with it. I haven't done an integration in POS yet, and I believe there is older code that just out of band measures elements with an empty environment, so we may need it for now.